src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
What is it used for?

Reply:
It is needed to mock `final` classes, `java.lang.ProcessBuilder` in our case.